test/db/cmd/charset

```diff
@@ -39,7 +39,7 @@ EOF
 EXPECT=<<EOF
 - offset -  A8A9 AAAB ACAD AEAF B0B1 B2B3 B4B5 B6B7  89ABCDEF01234567
 0x000019a8  c896 9381 4094 a495 8496 1500 0000 0000  ....@...........
-\xc8\x96\x93\x81@\x94\xa4\x95\x84\x96\x15
+
 - offset -  A8A9 AAAB ACAD AEAF B0B1 B2B3 B4B5 B6B7  89ABCDEF01234567
```

Comment:
Use psx or maybe it should be pszx :? So we can keep those escaped bytes i. The test

Reply:
Yes, done.